internal/ignition/ignition.go

```diff
@@ -496,6 +510,16 @@ func (g *installerGenerator) addBootstrapKubeletIpIfRequired(log logrus.FieldLog
 			return envVars, err
 		}
 		envVars = append(envVars, "OPENSHIFT_INSTALL_BOOTSTRAP_NODE_IP="+bootstrapIp)
+	} else if g.nodeIpAllocations != nil {
```

Comment:
g.nodeIpAllocations will always be part of MNO none platform right? 
In this case i am not sure that comment is right and i think better to change conditions in the function

Reply:
What comment ?

Reply:
https://github.com/openshift/assisted-service/pull/6257/files#diff-f0c8175849547a8e75a7442935a2ae99a16a72dfeee8efd9ea663fd2aa3c8c1bR504

Reply:
OK